setup

```diff
@@ -0,0 +1,84 @@
+whoisthis="Eric Harrison"  #your name, used for Git config
+youremail="ericmharrison@gmail.com"  #your email, used for Git config
+
+#update package list and upgrade all software
+sudo apt-get update 2>/dev/null && sudo apt-get -fy full-upgrade
+sudo apt -fy install software-properties-common apt-transport-https wget
+
+#check for git
+hasgit=$(apt list git --installed 2>/dev/null)
+if [ "$hasgit" == "Listing..." ]
+then
+  echo "Git's not installed! Let's fix that!"
+  sudo apt-get -fy install git
+fi
+
+#congratulate yourself
+echo
+echo "Git is installed: $(git --version)"
+
+#configure git
+git config --global --replace-all user.name "$whoisthis"
+git config --global user.email "$youremail"
+echo
+echo "Git User Name: $(git config --global --get user.name)"
+echo "Git User Email: $(git config --global --get user.email)"
+
+#add keys for new repositories
+echo "Adding keys"
+while read key
+do
+  echo $key
+  wget -q $key -O- | sudo apt-key add -
+done <keys
+
+#download packages from list in file
+while read url
+do
+  wget -nc $url
+done <files
+
+#install Go
+if [ ! -d "/usr/local/go" ]
+then
+  tar -xvf go1.14.4.linux-amd64.tar.gz
+  sudo chown -R root:root ./go
+  sudo mv go /usr/local
+  sudo echo "export GOPATH=usr/local/go" >> ~/.profile
+  sudo echo "export PATH=$PATH:/usr/local/go/bin:$GOPATH/bin" >> ~/.profile
+  source ~/.profile
+else
+  echo "Go is already installed"
+fi
+
+#install Kind
+if [ ! -f "/usr/local/go/kind" ]
+then 
+  GO111MODULE="on" go get sigs.k8s.io/kind@v0.8.1
```

Comment:
I think this env var may not be needed any more. Go modules should be on by default as of go 1.13something

Reply:
That's how it told me to do it here:
https://kind.sigs.k8s.io/docs/user/quick-start/